src/gross_to_net_generation.py

```diff
@@ -45,12 +45,15 @@ def convert_gross_to_net_generation(cems, eia923_allocated, plant_attributes, ye
                 "annual_subplant_ratio",
                 "annual_plant_ratio",
                 "annual_fuel_ratio",
+                "default_gtn_ratio",
             ]
         ],
         how="left",
         on=["plant_id_eia", "subplant_id", "report_date"],
         validate="m:1",
     )
+    # if there is a missing default gtn ratio, fill with 0.97
+    cems["default_gtn_ratio"] = cems["default_gtn_ratio"].fillna(0.97)
```

Comment:
Should we raise a warning here? Shouldn't all defaults come from  data/manual/default_gross_to_net_ratios?

Reply:
Yes good point. It would be good to flag where there are missing prime movers in the default gross to net ratios table